### ISSUE TITLE
[v2] #a11y

### ISSUE THREAD

**Issue by nicosomb** [wallabag/wallabag#1571]
I just read [Don’t use Slack?](https://medium.com/hacker-daily/don-t-use-slack-8e70452f3eed#.wltwsg7e8) about accessibility. 
It's a problem I already thought for wallabag. Few weeks ago, I asked @notabene and @OlivierNourry if they want to help us (and maybe they do :-) ). 

I just open this issue to not forget this important subject.

**Comment by notabene** [comment]
Sure, sure.
I added it to my TODO, for a start :wink:

**Comment by notabene** [comment]
Having been pinged on a social netork by @nicosomb 😉 I'll have to answer 5 years down the road.

The easiest solution (I guess), considering the complexity of wallabag, even considering the web version only, would be a workshop/hackathon of sorts to have back-and-forth sessions between a tester (off the top of my head me, but it could be someone else, and preferably a real disabled user) and a developer, and quick-iterate on features. I must say I have never worked on applications so far myself and would have yo read some documentation as to what should be expected.

**Comment by robinmetral** [comment]
Would love to help here (as a developer). Maybe a good way to start would be looking at user flows such as "sign up > save article > add tag > read article > star > archive"?
